app/Repositories/Summit/DoctrinePresentationActionTypeRepository.php

```diff
@@ -29,30 +33,41 @@ final class DoctrinePresentationActionTypeRepository
     /**
      * @inheritDoc
      */
-    protected function getBaseEntity()
+    protected function getBaseEntity(): string
     {
        return PresentationActionType::class;
     }
 
+    /**
+     * @param QueryBuilder $query
+     * @param Filter|null $filter
+     * @return QueryBuilder
+     */
+    protected function applyExtraJoins(QueryBuilder $query, ?Filter $filter = null): QueryBuilder
+    {
+        return $query->leftJoin("e.assigned_selection_plans", 'sp', Join::ON);
+    }
+
     /**
      * @return array
      */
-    protected function getFilterMappings()
+    protected function getFilterMappings(): array
     {
         return [
             'label' => 'e.label:json_string',
-            'summit_id' => new DoctrineLeftJoinFilterMapping("e.summit", "s" ,"s.id :operator :value")
+            'summit_id' => new DoctrineLeftJoinFilterMapping("e.summit", "s" ,"s.id :operator :value"),
+            'selection_plan_id' => new DoctrineLeftJoinFilterMapping("e.assigned_selection_plans", "sp" ,"sp.selection_plan_id :operator :value")
```

Comment:
u dont need this join
u are already doing the join at applyExtraJoins
just do
 'selection_plan_id' => 'sp.id'

Reply:
done